### PR TITLE
Convert South Carolina to pupa

### PR DESCRIPTION
**Pull request by brymut**
Hi, completed converting the South Carolina scrapper, it ran fine. 
Added a couple of DocStrings to the files I edited, hope this isn't an issue. Also opted to not write any tests since I wasn't sure of the testing strategies you normally use for the scrappers.

Thanks :)

Edit:
 Only completed the first part of the conversion process

### CODE REVIEW

**Comment by brymut** [comment]
Hi again,

Completed converting the South Carolina scrapper now, (Person, Bill and Event scrappers) making sure relevant information is intact. The scraper runs fine on my local docker container scraper. Also added docstrings to all the classes and methods in the files I edited.

Feel free to have a look @jamesturk @mileswwatkins 
Slack username : bryanmutai
Github username : @brymut 

Currently working on my proposal and will alert you when my draft is complete as well as a prototype demo of my idea. (Bill extraction, full text search, analysis and change detection)

File: openstates/sc/__init__.py [hunk 1]
```diff
@@ -1,83 +1,104 @@
...
+    """
+     Metadata containing information about state and sessions.
+     To be used by scrapers
+    """
+
+    division_id = "ocd-division/country:us/state:sc"
+    classification = "government"
+    name = "South Carolina"
+    url = "http://www.scstatehouse.gov/"
+    scrapers = {
```
Comment (jmcarp) [inline openstates/sc/__init__.py]: This should also include the updated event scraper.

File: openstates/sc/events.py [hunk 2]
```diff
@@ -58,6 +75,13 @@ def get_bill_description(self, url):
         return bill_description
 
     def scrape(self, chamber, session):
```
Comment (jmcarp) [inline openstates/sc/events.py]: Need to add default arguments here like you did for other scrapers.

File: openstates/sc/people.py [hunk 3]
```diff
@@ -88,11 +111,12 @@ def scrape(self, chamber, term):
                             '3rd V.C.': 'third vice-chair',
                             'Ex.Officio Member': 'ex-officio member',
                             'Chairman': 'chairman'}[role]
+
+                    person.extras['committee'] = committee
```
Comment (jmcarp) [inline openstates/sc/people.py]: Looks like `Person.add_membership` might be more appropriate here: https://github.com/opencivicdata/pupa/blob/master/pupa/scrape/popolo.py#L101

File: billy_metadata/sc.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                               '110 - (1993-1994)', '109 - (1991-1992)',
+                               '108 - (1989-1990)', '107 - (1987-1988)',
+                               '106 - (1985-1986)', '105 - (1983-1984)',
+                               '104 - (1981-1982)', '103 - (1979-1980)',
+                               '102 - (1977-1978)', '101 - (1975-1976)']
+
+)
+
+
+def session_list():
```
Comment (jmcarp) [inline billy_metadata/sc.py]: Could you move this to a method of the `SouthCarolina` class and rename it to `get_session_list`?

**Comment by jmcarp** [comment]
Could you add South Carolina to `flake8.sh` and check for any linter errors?

**Comment by brymut** [comment]
Hi again @jmcarp, made changes to address the points you brought up. Unless there's anything else you would like to add on, everything should be fine now.

**Comment by brymut** [comment]
Hi, sadly I won't be able to commit to working with you for summer of code this year but I have enjoyed learning about and working on this project and would like to keep on contributing during the summer. Even if it is just converting more scrappers to pupa. Thank you.

**Comment by jamesturk** [comment]
thanks for the check & sorry you won't be able to potentially join us for GSoC, starting to review output from this scraper today

**Comment by jamesturk** [comment]
merged w/ some rebasing

**Comment by brymut** [comment]
Thank you @jamesturk :) , will look into new stuff to work on. However, Is there a reason Github didn't note this pullrequest as merged?

**Comment by jamesturk** [comment]
because I pulled it into a branch to make a few small tweaks it didn't pick it up as the PR being merged- your commits are in master now though 👍

**Comment by brymut** [comment]
Alright, thanks again. 👍

**Pull request closed without merging**